File: 백준/Bronze/1159. 농구 경기/농구 경기.cc
#include<bits/stdc++.h>
using namespace std;
int n;
string s,ret;
int arr[26];

int main(){
	cin>>n;
	for(int i=0;i<n;i++){
		cin>>s;
		arr[s[0]-'a']++;
	}
	
	for(int i=0;i<26;i++){
//		cout<<char(i+'a')<<" : "<<arr[i]<<'\n';
		if(arr[i]>=5) ret+=char(i+'a');
	}
	if(ret.size())cout<<ret;
	else cout<<"PREDAJA";
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.java
// package baekjoon.week1.e1159;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine(); // 줄바꿈 제거 이게 무슨뜻이지?
        char[]arr = new char[26];
        for(int i=0;i<n;i++){
            String s = scanner.nextLine();
            //char a= s[0]-'a';
            char a = s.charAt(0);
            arr[a-'a']++;
        }
        String ret = "";
        for(int i=0;i<26;i++){
            if(arr[i]>4){
                ret+=(char)(i+'a');
            }
        }
        if (ret.length()>0) {
            System.out.println(ret);
        } else {
            System.out.println("PREDAJA");
        }
    }
}
